<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StemmedFileProcessingDialog</class>
 <widget class="QDialog" name="StemmedFileProcessingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="cbxReading">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reading stemmed documents from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxTfidf">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Counting tfidf indexes for words</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxWriting">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Writing to file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxDone">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>All done, you can close this window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
